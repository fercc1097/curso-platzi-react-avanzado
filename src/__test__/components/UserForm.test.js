import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import { UserForm } from '../../components/UserForm/index';

describe('<SubmitButton />', () => {
  test('Render del componente Footer ', () => {
    const userForm = mount(<UserForm />);
    expect(userForm.length).toEqual(1);
  });
});

describe('FavButton Snapshot', () => {
  test('Comprobar la Ui del componente Footer', () => {
    const userForm = create(<UserForm />).toJSON();
    expect(userForm).toMatchSnapshot();
  });
});
