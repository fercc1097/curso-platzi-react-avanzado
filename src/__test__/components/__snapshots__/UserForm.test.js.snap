// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavButton Snapshot Comprobar la Ui del componente Footer 1`] = `
<form
  className="sc-bdnylx gNCeED"
  onSubmit={[Function]}
>
  <h2
    className="sc-hKFyIo iRLQpE"
  />
  <input
    className="sc-gtssRu jZyEmI"
    onChange={[Function]}
    placeholder="Email"
    value=""
  />
  <input
    className="sc-gtssRu jZyEmI"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value=""
  />
  <button
    className="sc-jSFkmK eILAYh"
  />
</form>
`;
